File: 1.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Введите строку: ");
        String input = scanner.nextLine();
        int vowelCount = 0;
        int consonantCount = 0;
        input = input.toLowerCase();

        String vowels = "aeiouy";

        for (char c : input.toCharArray()) {
            if (Character.isLetter(c)) {
                if (vowels.indexOf(c) != -1) {
                    vowelCount++;
                } else {
                    consonantCount++;
                }
            }
        }

        if (vowelCount > consonantCount) {
            System.out.println("Да");
        } else if (vowelCount < consonantCount) {
            System.out.println("Нет");
        } else {
            System.out.println("Одинаково");
        }

        scanner.close();
    }
}

File: 1.swift
import Foundation
func analyzeString(_ input: String) {
    let vowels: Set<Character> = ["a", "e", "i", "o", "u", "y",
                                  "A", "E", "I", "O", "U", "Y"]
    let consonants: Set<Character> = Set("bcdfghjklmnpqrstvwxyzBCDFGHJKLMNPQRSTVWXYZ")
    
    var vowelCount = 0
    var consonantCount = 0
    for char in input {
        if vowels.contains(char) {
            vowelCount += 1
        } else if consonants.contains(char) {
            consonantCount += 1
        }
    }
    if vowelCount > consonantCount {
        print("Да")
    } else if vowelCount < consonantCount {
        print("Нет")
    } else {
        print("Одинаково")
    }
}

print("Введите строку:")
if let userInput = readLine() {
    analyzeString(userInput)
} else {
    print("Ошибка ввода.")
}



